src/odemis/acq/test/stream_test.py

```diff
@@ -2403,7 +2413,7 @@ def test_acq_arpol_leech(self):
         # check that number of angle resolved images is same as the total number of ebeam positions
         # include if multiple polarization images are required per ebeam position
         ar_das = sas.raw[1:-1]  # angle resolved data arrays
-        if sas._allPol.value:
+        if sas._acquireAllPol.value:
```

Comment:
Please remove entirely the `if` here, and just after the line setting the `are.polarization.value`, add a line setting `ars.acquireAllPol.value` to `False` (or `True`, if you prefer?).